include/ut.h

```diff
@@ -152,6 +152,7 @@ UT_test_t *UT_add_test( UT_test_suite_t *pSuite, const char *pTitle, UT_TestFunc
 #define UT_ASSERT_TRUE(value) CU_ASSERT_TRUE_FATAL(value)
 #define UT_ASSERT_FALSE(value) CU_ASSERT_FALSE_FATAL(value)
 #define UT_ASSERT_EQUAL(actual,expected) CU_ASSERT_EQUAL_FATAL(actual,expected)
+#define UT_ASSERT_EQUAL_NOT_FATAL(actual,expected) CU_ASSERT_EQUAL(actual,expected)
```

Comment:
Do you think we should add the full list?

e.g.
`UT_ASSERT_TRUE_NOT_FATAL`
`UT_ASSERT_FALSE_NOT_FATAL`

Reply:
Agreed. Should we also add UT_ASSERT_NOT_FATAL?

Reply:
Add NOT-FATAL equivalents to existing UT Test Macros as discussed